test/truffle/integration/backtraces/cext.rb

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved. This
+# code is released under a tri EPL/GPL/LGPL license. You can use it,
+# redistribute it and/or modify it under the terms of the:
+#
+# Eclipse Public License version 1.0
+# GNU General Public License version 2
+# GNU Lesser General Public License version 2.1
+
+require_relative 'backtraces'
+
+# Require some C ext to load C ext support
+require 'syslog'
+
+RB_FUNCALLV = Truffle::Interop.import('@rb_funcallv')
```

Comment:
Wow I never thought about doing this before

Reply:
More weird stuff coming :)